geoips/plugins/modules/readers/ami_netcdf.py

```diff
@@ -417,6 +407,13 @@ def _get_geolocation_metadata(metadata):
     geomet["roi_factor"] = 5  # roi = res * roi_factor, was 10
     geomet["num_lines"] = metadata["data"]["number_of_lines"]
     geomet["num_samples"] = metadata["data"]["number_of_columns"]
+    # Dynamically get offsets and scale factors needed for correct geolocation
+    # calculation
+    geomet["sub_lon"] = metadata["projection"]["sub_longitude"]
+    geomet["CFAC"] = np.abs(metadata["projection"]["cfac"])
+    geomet["COFF"] = np.abs(metadata["projection"]["coff"])
+    geomet["LFAC"] = np.abs(metadata["projection"]["lfac"])
+    geomet["LOFF"] = np.abs(metadata["projection"]["loff"])
```

Comment:
@evrose54 @lance-wilson This was the one that had some inconsistencies in the datafiles themselves, correct?  Is this where we get inconsistencies with upside down data reads?

Reply:
Yup. This is where that occurs. Previously we had hardcoded values for ``CFAC, COFF, LFAC, LOFF``, which is why we never ran into this problem before.

Reply:
So @lance-wilson had hard coded the values so the data was consistent with other datasets / readers ?  So now we are using the information in the data files themselves, which is either incorrect or just inconsistent with other datasets?  Do you need to create an issue for this (might already have one), link to this conversation, then link to the issue here from this conversation, then resolve this conversation?

Reply:
@mindyls As far as I remember, this was a case where the values (or maybe just one value?) within the data files had a different sign than the values in the manual for those files.  I'm not sure if that warrants an issue (I don't think we can do anything about it?)

Reply:
Maybe just the general issue for making sure the readers return data in a consistent orientation, which may or may not include resolving the sign issue on one of these values?

Reply:
Yeah, I think this can be handled in the xarray data standards. Just require that, for geostationary imagery, the 0th line is North and the -1th line is South (I think). Then just resolve that in whatever way is needed for each reader to ensure that they are oriented correctly.

Reply:
That is boring.

Reply:
See #817